charts/eol-exporter/values.yaml

```diff
@@ -0,0 +1,230 @@
+---
+eol-exporter:
+  enabled: true
+  namespaceOverride: ''
+  applicationName: eol-exporter
+  externalSecrets:
+    enabled: false
+  deployment:
+    enabled: true
+    image:
+      repository: saritasallc/eol-exporter
+      tag: prod-b9828d4
```

Comment:
LEt's add some links, where and how this docker image is built

Reply:
Done